social-network-api/src/test/java/fs/socialnetworkapi/entity/LikeTest.java

```diff
@@ -0,0 +1,35 @@
+package fs.socialnetworkapi.entity;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+
+class LikeTest {
+
+  @Test
+  void testGetUser() {
+    User user = mock(User.class);
+    Like like = new Like();
+    like.setUser(user);
+    assertEquals(user, like.getUser());
+  }
+
+  @Test
+  void testGetPost() {
+    Post post = mock(Post.class);
+    Like like = new Like();
+    like.setPost(post);
+    assertEquals(post, like.getPost());
+  }
+
+  @Test
+  void testConstructor() {
+    User user = mock(User.class);
+    Post post = mock(Post.class);
+    Like like = new Like(user, post);
+    assertNotNull(like);
+    assertEquals(user, like.getUser());
+    assertEquals(post, like.getPost());
```

Comment:
No need to test getters/setters/constructors. The main idea of the test is to set the initial state (setters/constructor) -> change the initial state (by a function that you are testing) -> get and compare the state with the expected ( using getters ). The getters, setters and constructors it's a default Java language operations, there is no logic that needs to be tested.

Reply:
It need for test cover 100%

Reply:
Why? As I remember you have 30% of coverage required for the new code by Sonar.